client/src/components/App.jsx

```diff
@@ -146,7 +146,7 @@ const App = () => {
         getOneProduct={getOneProduct}
         getStyles={getStyles}
         setCurrentProduct={setCurrentProduct}
-        currentProduct = {currentProduct}
+        currentProduct={currentProduct}
```

Comment:
This will be my new grammar hell lols

Reply:
lolol don't worry Nick, I'm here for you XD